### DISCUSSION TITLE
Is this client like official steam client?

### DISCUSSION THREAD

**Discussion by Hackerprod** [SteamRE/SteamKit#1215] [General]
Hello, why if I implement a client using this library do the other (official) steam clients not recognize me as Connected on the steam network?

**Comment by zxcvqwerasdf** [comment]
Yes, they can recognize that you auth from steamkit. But they don't care about it.

**Comment by Christian-Stieber** [comment]
Yeah, I'm late to the party, but new clients are "offline" by default.

You'll need to send a `CMsgClientChangeStatus` message to become visible.
